pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -22,17 +22,28 @@ import (
 	"github.com/gardener/gardener/pkg/operation/botanist/component"
 	"github.com/gardener/gardener/pkg/utils/managedresources"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
+	// LabelKey is the key of a label used for the identification of VPNShoot pods.
+	LabelKey = "app"
```

Comment:
There is a constant already in `v1beta1constants` (`LabelApp`) which you can use instead.